string state
bool   speaking
string grasped_item
